app/components/peopletable/peopletable.js

```diff
@@ -57,6 +60,7 @@ RemoveLinkCell.propTypes = {
   data: React.PropTypes.array,
   rowIndex: React.PropTypes.number,
   handleClick: React.PropTypes.func,
+  title: React.PropTypes.string,
```

Comment:
```suggestion
  title: React.PropTypes.string.isRequired,
```
Optional: It can be an opportunity to replace the deprecated React.PropTypes calls with the 'prop-types' package directly.

Reply:
prop-types package added